README.md

```diff
@@ -1,4 +1,48 @@
-HtUserRegistrationDoctrineORM
-=============================
-
-Doctrine2 ORM storage adapter for HtUserRegistration.
+HtUserRegistrationDoctrineORM
+=============================
+
+This module extends [HtUserRegistration](https://github.com/hrevert/HtUserRegistration)
+by replacing the Zend DB code with Doctrine.
+
+Install
+-------
+
+Require with composer
+```
+composer require hrevert/ht-user-registration-doctrine-orm
+```
+
+Include in config/application.config.php
+```php
+    'modules' => array(
+        ...
+        'HtUserRegistration',
+        'HtUserRegistrationDoctrineORM',  // include after HtUserRegistration
+        ...
+    ),
+```
+
+
+Configure
+----------
+
+Your User entity must implement `HtUserRegistrationDoctrineORM\Entity\UserInterface`.

```

Comment:
zfc-user requires ZfcUser\Entity\UserInterface https://github.com/ZF-Commons/ZfcUser/blob/1.x/src/ZfcUser/Entity/UserInterface.php and to implement what HtUserRegistrationDoctrineORM does the most complete solution is to have full database connections through Doctrine.  zf-oauth2-doctrine also provides this complete database navigation: https://github.com/TomHAnderson/zf-oauth2-doctrine/blob/master/src/Entity/UserInterface.php